Dependencies/Utilities/Sources/Utilities/Extensions/UIApplication+Extensions.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIApplication+Extensions.swift
+//  
+//
+//  Created by MaTooSens on 17/08/2023.
+//
+
+import UIKit
+
+public extension UIApplication {
+    static var rootViewController: UIViewController? {
```

Comment:
```suggestion
extension UIApplication {
    public static var rootViewController: UIViewController? {
```
żeby było spójnie z pozostałymi